packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -11,6 +22,8 @@ import './my-profile.scss';
 
 const MyProfile = observer(() => {
     const { general_store, my_profile_store } = useStores();
+    const [on_screen_keyboard, toggleOnScreenKeyboard] = React.useState(false);
+    const [show_real_name, toggleShowRealName] = React.useState(false);
```

Comment:
Same here, please use `should_` prefix and rename `toggleShowRealName` to something like `setShouldShowRealName`, also, real name is out of this card's scope.

Reply:
Real name is not on this card scope so I removed it